azure-toolkit-libs/azure-toolkit-appservice-lib/src/main/java/com/microsoft/azure/toolkit/lib/appservice/function/core/FunctionClass.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ */
+package com.microsoft.azure.toolkit.lib.appservice.function.core;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import java.util.List;
+
+public class FunctionClass implements IAnnotatable {
```

Comment:
FunctionAnnotationClass or AnnotationClass

Reply:
New pr https://github.com/microsoft/azure-maven-plugins/pull/1864 addressed the comments.